components/DashboardPage/components/HabitTracker/Habit/Habit.tsx

```diff
@@ -0,0 +1,45 @@
+import { Stack, Typography } from 'components';
+import { ChangeEventHandler, useState } from 'react';
+import { Checkbox } from '../Checkbox';
+import styles from './Habit.module.scss';
+
+interface HabitProps {
+  label: string;
+  days: boolean[];
+  onChange: any;
+  onChecked: any;
+}
+
+export function Habit({
+  label,
+  days,
+  onChange,
+  onChecked,
+}: HabitProps) {
+  return (
+    <Stack>
+      <Stack width="25%">
+        <input
+          type="text"
+          placeholder="Add a habit"
+          className={styles.Text}
+          onChange={onChange}
+          value={label}
```

Comment:
nit: should we have this kind of input configurable through our `Input` component?
the only differences is no label, no outline, no underline.

Reply:
I'll do it in a separate PR because we follow best practices around here `sheesh`